components/centraldashboard/public/ajax_test_helper.js

```diff
@@ -72,7 +72,13 @@ export function mockIronAjax(component, response, respondWithError = false) {
         component.dispatchEvent(
             new CustomEvent(finalEvent, eventPayload)
         );
-        return component.lastResponse || component.lastError;
+        const resp = component.lastResponse || component.lastError;
+
+        // So that code can await when using this dynamically
+        resp.completes = new Promise((res, rej) =>
+            (respondWithError?rej:res)(resp)
```

Comment:
*nit* Add spacing to comply with Google JavaScript horizontal whitespace guidelines.

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace